ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -432,6 +441,14 @@ def pytest_configure(config):
             check_clusters()
             if ocsci_config.RUN.get("cephcluster"):
                 gather_version_info_for_report(config)
+            if ocsci_config.MULTICLUSTER["acm_cluster"]:
```

Comment:
If we are just running deployment of OCP - will this try to collect also those details there, then it will probably fail as we don't have yet the deployment done yet?

If there will be any exception it will lead I guess to fail whole pytest as pytest_configure hook is very sensitive for whatever error causing pytest fail with Fatal error IIRC.

So we will need to check this please if it will cause any issue for whole deployment.

Reply:
the get_version function returns `None` if  the operator does not exist